sae/blocks.go

```diff
@@ -131,11 +131,13 @@ func (vm *VM) buildBlock(
 
 	state, err := worstcase.NewState(vm.hooks, vm.exec.ChainConfig(), vm.exec.StateCache(), lastSettled)
 	if err != nil {
-		log.Warn("Settled state not available")
+		log.Warn("Worst case state not able to be created",
+			zap.Error(err),
+		)
 		return nil, err
 	}
 
-	for _, b := range unsettledAncestry(parent, lastSettled.Height()) {
+	for _, b := range blocks.Range(lastSettled, parent) {
```

Comment:
```suggestion
	unsettled := blocks.Range(lastSettled, parent)
	for _, b := range unsettled {
```

As much as `blocks.Range()` simplifies everything, I think it's still useful to communicate the general intent, and will recommend the same below. Without this, we would force the reader to figure out the implication of the arguments to `Range()` whereas now they just need to confirm the correctness (or take it for granted).